apps/entities/models/entity.py

```diff
@@ -30,7 +30,7 @@
 
 NAME_MAX_LENGTH: int = 100
 
-TRUNCATED_DESCRIPTION_LENGTH: int = 1200
+TRUNCATED_DESCRIPTION_LENGTH: int = 250
```

Comment:
@sunzae99, @iacobfred can you confirm that you did not have another purpose in mind when you originally set this limit?

Reply:
Nope, I'm fine with this.